ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -220,7 +226,7 @@ echo "MIRROR_PROXY_REGISTRY_QUAY: ${MIRROR_PROXY_REGISTRY_QUAY}"
 MIRROR_PROXY_REGISTRY=`echo "${MIRROR_REGISTRY_HOST}" | sed 's/5000/6002/g' `
 echo "MIRROR_PROXY_REGISTRY: ${MIRROR_PROXY_REGISTRY}"
 set_cluster_auth
-enable_qe_proxy_registry
+set_CA_for_nodes
```

Comment:
Per our last discussion, we do not need this in this ref step, right?

Reply:
Actually, we need it. The CA should be configed for each node so that the image can be pulled successfully.
```yaml
# Slove: x509: certificate signed by unknown authority
# Config CA for each cluster node so that it can pull images successfully.
# It not only be used by Sample operator, but ICSP and all pods that pulling images from the external registry.
# $ oc debug node/qe-daily-0721-p79bt-worker-3-knbkr
# sh-4.4# chroot /host
# sh-4.4# ls -l /etc/pki/ca-trust/source/anchors/
# total 0
# -rw-------. 1 root root 0 Jul 20 23:30 openshift-config-user-ca-bundle.crt
# sh-4.4# ls -l /etc/docker/certs.d
# total 0
# drwxr-xr-x. 2 1001 root 20 Jul 20 23:30 image-registry.openshift-image-registry.svc.cluster.local:5000
# drwxr-xr-x. 2 1001 root 20 Jul 20 23:30 image-registry.openshift-image-registry.svc:5000
# drwxr-xr-x. 2 1001 root 20 Jul 21 03:30 vmc.mirror-registry.qe.devcluster.openshift.com:5000
# drwxr-xr-x. 2 1001 root 20 Jul 21 03:30 vmc.mirror-registry.qe.devcluster.openshift.com:6001
# drwxr-xr-x. 2 1001 root 20 Jul 21 03:30 vmc.mirror-registry.qe.devcluster.openshift.com:6002
```